pkg/utils/etcdutil/etcdutil.go

```diff
@@ -440,6 +458,13 @@ func (checker *healthyChecker) update() {
 			}
 			continue
 		}
+
+		if status, ok := checker.isUnHealthy.Load(ep); ok && status.(bool) {
+			log.Info("some etcd server is unhealthy", zap.String("endpoint", ep))
+			// set unhealthy etcd client to healthy
+			checker.isUnHealthy.Delete(ep)
```

Comment:
If we remove `isUnHealthy` here, will we receive "frequently changed" error again？

Reply:
trend check will make sure this node not participate in campaigning leader